androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -110,6 +118,8 @@ fun EnhancedVideoPlayer(
         }
         exoPlayer.addListener(listener)
 
+        exoPlayer.addAnalyticsListener(EventLogger())
```

Comment:
Why is there this analysis logger?

Reply:
To log the tracks for debug. Should we add it just when testing or add a flag to toggle it?

Reply:
The way it is implemented, this will be enabled for all build flavors and have performance implications. Ideally, leave this to whoever is implementing and want more info to debug

Reply:
Makes sense!